### PR TITLE
Issue #217 : Added Implementation of Air-Cargo-problem Fig 10.1 for AIMA3e branch

### PR DESCRIPTION
**Pull request by divsidhu-26**
Please check if the solution is correct. Also I wasnt sure if i have to write comments to explain the code. Please update me on that and also tell any changed in the code if they were to be made. Isssue #217

### CODE REVIEW

File: aima-core/src/main/java/aima/core/learning/knowledge/Classical_Planning/AirCargoProblem.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	}
+}
+class Plane{
+	String name;
+	Airport airport;
+	Plane(String p){
+		this.name = p;
+	}
+	public boolean equals(Plane a){
+    	if((this.name).equals(a.name) && (this.airport).equals(a.airport)){
```
Comment (duchuyctlk) [inline aima-core/src/main/java/aima/core/learning/knowledge/Classical_Planning/AirCargoProblem.java]: How about directly returning the result? 😄 

```java
return this.name.equals(a.name) && this.airport.equals(a.airport);
```

File: aima-core/src/main/java/aima/core/learning/knowledge/Classical_Planning/AirCargoProblem.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+		System.out.println(at(C1,JFK)+" "+at(C2,SFO));
+	}
+}
+class Airport{
+	String name;
+    Airport(String s){
+    	this.name = s;
+    }
+    public boolean equals(Airport a){
+    	if((this.name).equals(a.name)){
```
Comment (duchuyctlk) [inline aima-core/src/main/java/aima/core/learning/knowledge/Classical_Planning/AirCargoProblem.java]:
```java
return this.name.equals(a.name);
```

😄

File: aima-core/src/main/java/aima/core/learning/knowledge/Classical_Planning/AirCargoProblem.java (hunk 1 above)
Comment (divsidhu-26) [reply aima-core/src/main/java/aima/core/learning/knowledge/Classical_Planning/AirCargoProblem.java]: yea sure and do i need to comment explanations along with the code

Comment (duchuyctlk) [reply aima-core/src/main/java/aima/core/learning/knowledge/Classical_Planning/AirCargoProblem.java]: Oh, this approach is really clear already, i don't think that explanation required here.

Comment (divsidhu-26) [reply aima-core/src/main/java/aima/core/learning/knowledge/Classical_Planning/AirCargoProblem.java]: ok i will erase the comments............

**Comment by ctjoreilly** [comment]
This is a neat little example. Unfortunately, it takes an imperative programming approach to what is intended to be a declarative approach (i.e. PDDL). The ideal way to tackle this problem is to have a specialized parser based on the FOL one, that knows how to extract out the Action, PRECOND, and Effect sections - validate they are of the correct form and make them available to the caller. The Air Cargo example itself then would just be a text file mirroring Figure 10.1 exactly.

**Comment by divsidhu-26** [comment]
ok i will try to do it with that approach..........

**Pull request closed without merging**